common/src/main/java/org/opensearch/ml/common/dataset/remote/RemoteInferenceInputDataSet.java

```diff
@@ -23,11 +24,17 @@ public class RemoteInferenceInputDataSet extends MLInputDataset {
 
     @Setter
     private Map<String, String> parameters;
+    private ActionType actionType;
 
     @Builder(toBuilder = true)
-    public RemoteInferenceInputDataSet(Map<String, String> parameters) {
+    public RemoteInferenceInputDataSet(Map<String, String> parameters, ActionType actionType) {
         super(MLInputDataType.REMOTE);
         this.parameters = parameters;
+        this.actionType = actionType;
```

Comment:
add to writeTo method  and `public RemoteInferenceInputDataSet(StreamInput streamInput) throws IOException {`?